docs/getting_started/explorer.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Explorer
+sidebar_position: 6
+---
+
+From your  Calimero [Console](https://app.calimero.network/dashboard) you can access the explorer which provides you with the ability to view all crypto transactions and information including: accounts, blocks, transactions and receipts, Fungible Tokens and Non-Fungible Tokens in details. 
+
+It is by default a fully private explorer only available to the administrators of the network, but specific information can be shared via API endpoints or shared links through the permissions management service implemented inside the Console.
+
+![](../../static/img/explorer.png)
+
+## Account
+
+Accounts store digital assets, maintain balances, and track transactions.
+
+![](../../static/img/demo_account.png)
+
+By clicking on any of the account ID, you'll be able to have an overview of:
+
+- The number of transactions occured on that account
+- The amount of storage used
+- The native account balance currently on the account
+- The amount of validator stake used
```

Comment:
Check out the description of validator stake on NEAR explorer ([link](https://explorer.testnet.near.org/accounts/calimero.testnet))